### PR TITLE
Reflect deprecation of the HTML renderer on image CORS details

### PR DESCRIPTION
**Pull request by JaffaKetchup**
_Description of what this PR is changing or adding, and why:_

Removes references to the HTML renderer (as it is being deprecated), except for a small version note. Re-organised the page to better show the reduced information load and remove redundant headings.

I believe this comes into effect with 3.25.0, at the same time as https://github.com/flutter/flutter/pull/152683 is released, but https://github.com/flutter/website/pull/11045 has already been made, so this PR may be suitable immediately.

_Issues fixed by this PR (if any): #11154

## Presubmit checklist

- [x] This PR is marked as draft with an explanation if not meant to land until a future stable release.
- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

**Review by sfshaza2 — APPROVED** [review]
Smoothed out a sentence that was a bit hard to parse. Otherwise, LGTM. Thx!

File: src/content/platform-integration/web/web-images.md [hunk 1]
```diff
@@ -4,13 +4,16 @@ short-title: Web images
 description: Learn how to load and display images on the web.
 ---
 
-The web supports the standard [`Image`][] widget to display images.
+The web supports the standard [`Image`][] widget and the more 
+advanced [`dart:ui/Image`][] class (where more fine-grained control 
+is needed to display images.
```
Comment (parlough) [inline src/content/platform-integration/web/web-images.md:9]:
```suggestion
advanced [`dart:ui/Image`][] class (when more fine-grained control 
is needed to display images).
```

**Review by parlough — APPROVED** [review]
Thanks so much for these adjustments!

I expect more changes will be needed in the future with Harry's work on https://github.com/flutter/flutter/issues/149843, but it's nice to see some simplification to the docs in the mean time :D
